src/routes/assets.router.js

```diff
@@ -32,10 +32,13 @@ router.get('/order', async (req, res) => {
 })
 
 router.get('/consume', async (req, res, next) => {
-  const responseObj = await assetsController.GET_consume(req, res)
-
-  console.log('Files consumed')
-  res.status(200).json(responseObj)
+  if (res.locals.globaloptions.enableconsume) {
+    const responseObj = await assetsController.GET_consume(req, res)
+    console.log('Files consumed')
+    res.status(200).json(responseObj)
+  } else {
+    res.status(200).json('Consuming is not allowed.')
```

Comment:
```suggestion
    res.status(405).json('Consuming is not allowed.')
```

`405: Method not allowed` seems like the most appropriate status code here https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405